pkg/counter/prefixes.go

```diff
@@ -50,10 +50,12 @@ func createIPNet(ones, bits int) netip.Prefix {
 
 // DefaultPrefixLengthCounter creates a default prefix length counter that
 // already counts the minimum and maximum prefix lengths for IP hosts and
-// default routes (ie, /32 and /0). As with NewPrefixLengthCounter, inesrtions
+// default routes (ie, /32 and /0). As with NewPrefixLengthCounter, insertions
 // are limited to the specified maximum number of unique prefix lengths.
-func DefaultPrefixLengthCounter(maxUniquePrefixes6, maxUniquePrefixes4 int) *PrefixLengthCounter {
-	counter := NewPrefixLengthCounter(maxUniquePrefixes6, maxUniquePrefixes4)
+func DefaultPrefixLengthCounter() *PrefixLengthCounter {
+	maxIPv4 := net.IPv4len*8 + 1
```

Comment:
not a point, but where does the +1 come from?

Reply:
There's 4 Bytes in an IPv4 address, times 8 bits in a byte, plus one to represent the mask of 0. `w.x.y.z/0`, `w.x.y.z/1`, ... `w.x.y.z/32` starts at 0 and reaches /32, the full width of an IP Prefix.